network-api/networkapi/mozfest/templates/fragments/hero/large.html

```diff
@@ -1,23 +1,47 @@
-{% load wagtailcore_tags wagtailimages_tags %}
+{% load wagtailcore_tags wagtailimages_tags i18n %}
 
 <div class="background-wrapper">
   <div class="overlay"></div>
-  {% if page.banner %}
+  {% if banner_type == "video" %}
+    <video class="banner-video" playsinline autoplay muted loop preload poster="" >
```

Comment:
we may want to take off `autoplay` and instead trigger a `video.play()` (and label reveal) only after we know that the play/pause icons have fully loaded, otherwise the video can start playing before the controls are ready.

Reply:
Follow-up filed: https://github.com/mozilla/foundation.mozilla.org/issues/4309